src/stage-release.sh

```diff
@@ -26,13 +26,17 @@ if [ $# -eq 0 ]; then
 fi
 
 DEST_PATH=$1
+DEST_PATH="$(cd "$DEST_PATH" && pwd)"
 
 pushd $(dirname "$0")
 PULSAR_PATH=$(git rev-parse --show-toplevel)
 VERSION=`./get-project-version.py`
 popd
 
-cp $PULSAR_PATH/target/apache-pulsar-$VERSION-src.tar.gz $DEST_PATH
+pushd "$(dirname "$0")/.."
+git archive --format=tar.gz --output="$DEST_PATH/apache-pulsar-$VERSION-src.tar.gz" --prefix="apache-pulsar-$VERSION-src/" HEAD
```

Comment:
> HEAD

Perhaps use "v$VERSION", but I use `HEAD` for testing and as long as the release tag is stable and we release on the tag, it doesn't cause any difference.